cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -543,6 +543,30 @@ func (s *Server) GetInstalledPackageSummaries(ctx context.Context, request *core
 		installedPkgSummaries[i] = installedPkgSummaryFromRelease(r)
 	}
 
+	// Fill in the latest package version for each.
+	// TODO(mnelson): Update to do this with a single query rather than iterating and
+	// querying per release.
```

Comment:
You mean adding a new method in the `PostgresAssetManager` and delegate the complexity of an actual SQL query, don't you?
If so, yep, I agree. We better perform this kind of operations on the DB side.

BTW, we should refactor it at some point (using an ORM or sth)

Reply:
> You mean adding a new method in the `PostgresAssetManager` and delegate the complexity of an actual SQL query, don't you?
> If so, yep, I agree. We better perform this kind of operations on the DB side.
> 
> BTW, we should refactor it at some point (using an ORM or sth)

If we end up using postgresql long term, then yes, it'd be good to abstract back a bit to an ORM, but that depends on the outcome of the plugin caching etc. So I see no use to invest in that until we know.